doc/orchestrator.rst

```diff
@@ -63,108 +326,207 @@ on that compute host.
+configure the the distribution of computational tasks (``RunSettings.set_nodes()``). In this
+example, we specify to SmartSim that we intend the Model to run on a single compute node.
 
-Redis
-=====
+.. code-block:: python
 
-.. _Redis: https://github.com/redis/redis
-.. _RedisAI: https://github.com/RedisAI/RedisAI
+    # Initialize a RunSettings object
+    model_settings = exp.create_run_settings(exe=exe_ex, exe_args="/lus/scratch/richaama/clustered_model.py")
+    # Configure RunSettings object
+    model_settings.set_nodes(1)
 
-The ``Orchestrator`` is built on `Redis`_. Largely, the job of the ``Orchestrator`` is to
-create a Python reference to a Redis deployment so that users can launch, monitor
-and stop a Redis deployment on workstations and HPC systems.
+Step 2: Initialize
+""""""""""""""""""
+Next, create a ``Model`` instance using the ``Experiment.create_model()``.
+Pass the ``model_settings`` object as an argument
+to the ``create_model()`` function and assign to the variable ``model``.
+.. code-block:: python
 
-Redis was chosen for the Orchestrator because it resides in-memory, can be distributed on-node
-as well as across nodes, and provides low latency data access to many clients in parallel. The
-Redis ecosystem was a primary driver as the Redis module system provides APIs for languages,
-libraries, and techniques used in Data Science. In particular, the ``Orchestrator``
-relies on `RedisAI`_ to provide access to Machine Learning runtimes.
+    # Initialize a SmartSim Model
+    model = exp.create_model("colo_model", model_settings)
 
-At its core, Redis is a key-value store. This means that put/get semantics are used to send
-messages to and from the database. SmartRedis clients use a specific hashing algorithm, CRC16, to ensure
-that data is evenly distributed amongst all database nodes. Notably, a user is not required to
-know where (which database node) data or Datasets (see Dataset API) are stored as the
-SmartRedis clients will infer their location for the user.
+Step 2: Colocate
+""""""""""""""""
+To colocate the model, use the ``Model.colocate_db_tcp()`` function.
+This function will colocate an Orchestrator instance with this Model over
+a Unix domain socket connection.
 
+.. code-block:: python
 
-KeyDB
-=====
+    # Colocate the Model
+    model.colocate_db_tcp()
 
-.. _KeyDB: https://github.com/EQ-Alpha/KeyDB
+Step 3: Start
+"""""""""""""
+Next, launch the colocated model instance using the ``Experiment.start()`` function.
+.. code-block:: python
 
-`KeyDB`_ is a multi-threaded fork of Redis that can be swapped in as the database for
-the ``Orchestrator`` in SmartSim. KeyDB can be swapped in for Redis by setting the
-``REDIS_PATH`` environment variable to point to the ``keydb-server`` binary.
+    # Launch the colocated Model
+    exp.start(model, block=True, summary=True)
 
-A full example of configuring KeyDB to run in SmartSim is shown below
+test
 
-.. code-block:: bash
+Cleanup Experiment
+------------------
```

Comment:
Can we put an explanation here that you don't need to stop anything like before with the clustered deployment because the colocated deployment terminates with the ``Model`` resource job completing.